lms/templates/fields/field_image.underscore

```diff
@@ -0,0 +1,15 @@
+<div class="image-wrapper">
+    <img class="image-frame" src="<%- gettext(imageUrl) %>"  alt="<%=imageAltText%>"/>
+    <div class="u-field-actions">
+        <a class="u-field-upload-button">
+            <div class="upload-button-icon" aria-hidden="true"><%= uploadButtonIcon %></div>
```

Comment:
I noticed on the sandbox that there as screen reader text here for the icon.  It's unncessary since there is accessible text included in the uploadButtonTitle below.  Also, the screen reader text was actually inaccessible because its parent had aria-hidden=true on it, which makes the text inaccessible.